<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoDownTheStairs</class>
 <widget class="QMainWindow" name="GoDownTheStairs">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoDownTheStairs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_pause">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_restart">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string>重新开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>层数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_p1_life">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LIFE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_p2_life">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LIFE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_music">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>music</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
